js/country.js

```diff
@@ -1,44 +1,38 @@
-///////////////////////////////////      APIs
+// my code
+let country = localStorage.getItem("country");
+console.log(country)
+const containerDiv = document.getElementsByClassName("h-photo")[0];
 
-//Pictures API
 const pixaKey = "24477339-22b8ee8a367b2cdaebd16eb69";
 const pixaAPI = fetch(
-  `https://pixabay.com/api/?key=${pixaKey}&q=paris&image_type=photo&pretty=true&imageHeight=1080&imageWidth=1920`
-);
-
-pixaAPI.then((data) => console.log(data.json()));
-
-//Weather API
-const weatherAPI = fetch(`https://goweather.herokuapp.com/weather/paris`);
-
-weatherAPI.then((data) => console.log(data.json()));
+  `https://pixabay.com/api/?key=${pixaKey}&q=${country}&image_type=photo&pretty=true&imageHeight=1080&imageWidth=1920`
+)
+  .then((res) => {
```

Comment:
write some description about what these lines do, what the input and output ... [clean code]